docs/source/installation.rst

```diff
@@ -214,6 +219,24 @@ your environment is activated and linked to whatever IDE you are using.  If you
 Notebooks, follow `these instructions <https://janakiev.com/blog/jupyter-virtual-envs/>`_ for
 adding your virtual environment as a new kernel for your notebook.
 
+Dependency error on mac ARM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are using a mac with an ARM processor, you may encounter an error when installing
+``sktime[all_extras]``.  This is due to the fact that some libraries included in ``all_extras``
+are not prepared to run in ARM-based processors.
+
+If you want to still install all the remaining compatible libraries from ``all_extras`` you need
```

Comment:
I would say more clearly: the workaround is to not install some of the packages, and install Mac compatible replacements for others.